lua/cmp/types/cmp.lua

```diff
@@ -152,6 +152,7 @@ cmp.ItemField = {
 ---@field public keyword_length integer|nil
 ---@field public max_item_count integer|nil
 ---@field public group_index integer|nil
+---@field public entry_filter function|nil
```

Comment:
You should add type annotation here.

Reply:
See: [a7d6dd2](https://github.com/hrsh7th/nvim-cmp/pull/1067/commits/a7d6dd29c6ec3acd7bee514c4291aa5a062648f8)